.travis.yml

```diff
@@ -1,19 +1,19 @@
+os: "linux"
+dist: "xenial"
 language: node_js
+node_js:
+  - 10
+  - 12
+  - 14
 before_install:
-  # Create a master branch for commitlint
   # https://github.com/conventional-changelog/commitlint/issues/6
   - git remote set-branches origin master && git fetch
 jobs:
   include:
-    # Define the release stage that runs semantic-release
     - stage: release
-      node_js:
-        - 10
-        - 12
+      node_js: lts/*
       deploy:
         on:
           branch: master
         provider: script
-        skip_cleanup: true
```

Comment:
should this be `cleanup: false` now ?

Reply:
Nope, https://blog.travis-ci.com/2019-08-27-deployment-tooling-dpl-v2-preview-release